lambda-loader/template.yaml

```diff
@@ -111,7 +111,38 @@ Resources:
           AttributeName: "Region"
           KeyType: "RANGE" # Sort Key
       TableName: yesterday-rating-record-table
-      
+  ChannelTable:
```

Comment:
What is the advantage of making the table with the `templete.yml` instead of from the python interface?

Reply:
Separation of code. All the infrastructure can be found in template.yml and the business logic in the python file.

Reply:
Do you need python code to create the table locally?